src/solver/gemm_bwd.cpp

```diff
@@ -43,6 +43,7 @@
 MIOPEN_DECLARE_ENV_VAR(MIOPEN_CONV_PRECISE_ROCBLAS_TIMING)
 
 #define WORKAROUND_MIOPENGEMM_ISSUE_59 1
+#define WORKAROUND_MIOPENGEMM_ISSUE_2474 1
```

Comment:
@carlushuang  It's likely unnecessary since in the release dockers we are not seeing these issues.

Reply:
I should not push that commit related to bwd-gemm on my local machine. Now that commit is reverted